tsdb/series_index.go

```diff
@@ -19,7 +19,11 @@ const (
 )
 
 const (
-	SeriesIndexElemSize   = 16 // offset + id
+	// SeriesIDSize is the size in bytes of a series key ID.
+	SeriesIDSize        = 8
+	SeriesOffsetSize    = 8
+	SeriesIndexElemSize = 16 // offset + id
```

Comment:
Maybe `SeriesIndexElemSize = SeriesOffsetSize + SeriesIDSize` then we don't need the comment.